lib/init.coffee

```diff
@@ -10,4 +15,42 @@ module.exports =
         type: 'string'
 
   activate: ->
-    console.log 'activate linter-gjslint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-gjslint.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'gjslint'
+      grammarScopes: ['source.js', 'source.js.jquery', 'text.html.basic']
+      scope: 'file'
+      lintOnFly: true
+      lint: (editor) =>
+        filePath = editor.getPath()
+        cwd = path.dirname(filePath)
+        tempFile path.basename(filePath), editor.getText(), (tmpFilePath) =>
```

Comment:
https://github.com/google/closure-linter/issues/96
We don't have to wait for it